src/Tribe/Template.php

```diff
@@ -672,10 +801,13 @@ public function template( $name, $context = [], $echo = true ) {
 			$namespace        = array_merge( $origin_folder_appendix, $legacy_namespace );
 		}
 
-		// Setup the Hook name
+		// Setup the Hook name.
 		$legacy_hook_name = implode( '/', $legacy_namespace );
 		$hook_name        = implode( '/', $namespace );
 
+		// Store the current hookname for the purposes of entry-points.
+		$this->set_template_current_hook_name( $hook_name );
```

Comment:
Just curious, if a template is outputting another template, does that prevent being able to hook into the end of the parent template?

Reply:
Oh yeah that is a good question I will poke around that behavior. :real_heart:

Reply:
Modified the behavior here, to make sure it saves the previous locally and replaces before exiting. So it seems to do the job of having the entry points correctly even when you have child templates.

Reply:
👍